src/Farmer/Arm/BingSearch.fs

```diff
@@ -0,0 +1,23 @@
+[<AutoOpen>]
+module Farmer.Arm.BingSearch
+
+open Farmer
+
+let accounts = ResourceType ("Microsoft.Bing/accounts", "2020-06-10")
+[<Literal>]
+let private kind = "Bing.Search.v7"
+
+type Accounts =
+    { Name: ResourceName
+      Location: Location
+      Sku: BingSearch.Sku
+      Tags: Map<string,string>
+      Properties: {| statisticsEnabled: bool |} }
```

Comment:
Just move statistics to the top level. Consider replacing bool with Enabled / Disabled.

Reply:
The ARM type shouldn't be a 1:1 mapping with the JSON but an F# view on it.

Reply:
Good hint, fixed : )